scripts/build/testdata/build_all_except_host.txt

```diff
@@ -551,67 +551,72 @@ cmake -S {root}/examples/shell/mbed -B {out}/mbed-CY8CPROTO_062_4343W-shell-rele
 # Generating nrf-native-posix-64-tests
 bash -c 'source "$ZEPHYR_BASE/zephyr-env.sh";
 export GNUARMEMB_TOOLCHAIN_PATH="$PW_PIGWEED_CIPD_INSTALL_DIR";
-west build --cmake-only -d {out}/nrf-native-posix-64-tests -b native_posix_64 {root}/src/test_driver/nrfconnect'
+west build --cmake-only -d {out}/nrf-native-posix-64-tests -b native_posix_64 {root}/src/test_driver/nrfconnect --'
```

Comment:
the -- seems useless here. Can you modify the script to not have unneeded arguments?

Reply:
Updated. Also noticed that the test data I previously sent in were bad, regenerated now.